eslintrc

```diff
@@ -1,155 +0,0 @@
-# ESLint documentation can be found at http://eslint.org/docs/
```

Comment:
No, we should leave the eslintrc and update it as needed so we can run locally as usual. There are plenty of files that have been untouched for almost a year and probably won't be touched for a long time.